addons/binding/org.openhab.binding.nest.test/META-INF/MANIFEST.MF

```diff
@@ -1,26 +1,44 @@
-Manifest-Version: 1.0
-Bundle-ManifestVersion: 2
-Bundle-Name:  Nest Binding Tests
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Bundle-SymbolicName: org.openhab.binding.nest.test;singleton:=true
-Bundle-Vendor: openHAB
-Bundle-Version: 2.3.0.qualifier
-Export-Package: org.openhab.binding.nest;uses:="org.eclipse.smarthome.te
- st"
-Fragment-Host: org.openhab.binding.nest
-Import-Package: 
- org.eclipse.jdt.annotation;resolution:=optional,
- org.eclipse.smarthome.core.common.registry,
- org.eclipse.smarthome.core.events,
- org.eclipse.smarthome.core.thing.util,
- org.eclipse.smarthome.test.java,
- org.eclipse.smarthome.test.storage,
- org.hamcrest;core=split,
- org.junit,
- org.mockito,
- org.mockito.stubbing,
- org.openhab.binding.nest,
- org.openhab.binding.nest.handler,
- org.osgi.framework,
- org.osgi.service.device,
- org.slf4j
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ManifestVersion: 2
+Bundle-Name: Nest Binding Tests
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.nest.test;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Export-Package: 
+ org.openhab.binding.nest;uses:="org.eclipse
```

Comment:
Just curious this line is wrapped, but the line 6 is longer, is there a reason to wrap this one but not line 6

Reply:
The reason is I copied it from the Tradfri test binding [manifest](https://github.com/eclipse/smarthome/blob/936287de21b7a6baabcf969899b9b62d2faac0bf/extensions/binding/org.eclipse.smarthome.binding.tradfri.test/META-INF/MANIFEST.MF#L9) where it needs to be wrapped. So you are right that the wrapping does not make any sense here. :smile: 

At the time the Tradfri binding was one of the few bindings I could find that had working OSGi tests that could be run from Eclipse. So I quickly tried a lot to get the Nest binding OSGi tests also working. The cleanup could have been a bit better!